authors/apps/authentication/backends.py

```diff
@@ -1,13 +1,49 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
-
-"""Configure JWT Here"""
-class JWTAuthentication:
-    def authenticate(self, another):
-        pass
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from .models import User
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    """ This validate the token."""
+
+    # initializing the token as an empty string
+    keyword = 'Token'
+
+    def authenticate(self, request):
+        request.user = None
+
+        # returns Authorization header
+        # as a bytestring
+        auth_header = authentication.get_authorization_header(request).split()
+
+        if not auth_header or auth_header[0].lower() != self.keyword.lower():
+            return None
```

Comment:
Return message is no descriptive.

Reply:
please give me suggestions on this

Reply:
This is alright. The return statement should be None